src/App.js

```diff
@@ -1,12 +1,15 @@
 import "./App.css";
 import Navbar from "./components/Navbar";
```

Comment:
You can add a index.js file and export all components from there so you can import it like this.
```
import {Footer, Navbar,Sidebar} from "./components"
```
This makes the code look more clean

Reply:
sure will do that!